model/clusters_mgmt/v1/add_on_version_type.model

```diff
@@ -35,6 +35,9 @@ class AddOnVersion {
 	// List of sub operators for this add-on version.
 	SubOperators []AddOnSubOperator
 
+	// Config for the addon
```

Comment:
Can you expand on this comment, a little more, it makes sense to me as I have context of what this `config` is but for someone with no context this is a bit abstract.